README.md

```diff
@@ -692,6 +698,7 @@ Brazilian Portugese version : [here](https://github.com/LewisVo/Awesome-Linux-So
 - [GnuPG](https://www.gnupg.org/) - Gnu Privacy Guard
 - [GuFW](http://gufw.org/) - One of the easiest firewalls in the world of Linux. [![Open-Source Software][OSS Icon]](https://code.launchpad.net/gui-ufw)
 - [IPrange](https://github.com/firehol/iprange) - A very fast command line utility for processing IP lists (merge, compare, exclude, etc). [![Open-Source Software][OSS Icon]](https://github.com/firehol/iprange)
+- [keysniffer](https://github.com/jarun/keysniffer) - Linux kernel mode debugfs keylogger. [![Open-Source Software][OSS Icon]](https://github.com/jarun/keysniffer)
```

Comment:
This is unrelated to the repo but I hope you package this tool instead of let user do the "make".

Reply:
I understand. As it is a kernel module, the way to go is `dkms` integration rather than packaging (the way we know it in userland). On the other hand, I have kept it intentionally hard to use, decipher and camouflaged so people who _really_need it actually use it. ;)